test/stdlib/NSObject.swift

```diff
@@ -263,3 +262,8 @@ print( // CHECK-NEXT: true
   _getSuperclass(_getSuperclass(E.self)!) == NSObject.self)
 print( // CHECK-NEXT: true
   _getSuperclass(_getSuperclass(_getSuperclass(E.self)!)!) == nil)
+
+print("NSObject's type id")
+print(CFGetTypeID(NSObject()))
+// CHECK: NSObject's type id
+// CHECK-NEXT: 1
```

Comment:
Ah, it's a Swift object's type ID that you care about, right?

Reply:
This is basically assert(CoreFoundation didn't change something unexpectedly)